ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/host/Webserver.java

```diff
@@ -70,9 +73,8 @@ private void handleRequest(Socket socket) throws IOException {
 
   private ServerSocket createServerSocket() {
     try {
-      return new ServerSocket(0);
-    }
-    catch (IOException e) {
+      return new ServerSocket(0, 200, InetAddress.getLoopbackAddress());

```

Comment:
Why do we need to pass the backlog size and the loopback address?

Reply:
This is actually part of my original solution attempt for the webserver issue. Didn't really solve the issue by itself. The n we applied your solution proposal on top of this, forgot to redo this line.